### ISSUE TITLE
Doors

### ISSUE THREAD

**Issue by TyHen** [GreatOrator/FallCraft-Map#24]
Im sorry GreatOrator and Teagan75, but I think I may have found something else. I don't want to break your map or anything but some of the doors look like bikes to me which can be passed through, but others look like tables which can not be passed through. So far this is in Pleasant Valley. This is not a game changer, and I know what it is like to make something and test constantly as I am a programmer myself. Again Thank you for the awesome modpacks.
![2018-05-26_09 13 51](https://user-images.githubusercontent.com/31518304/40576493-8aa0144c-60c5-11e8-8dc6-0e0b3ba88d6c.png)
![2018-05-26_09 14 09](https://user-images.githubusercontent.com/31518304/40576495-8ab6d24a-60c5-11e8-969c-39990db5f0c0.png)

**Comment by GreatOrator** [comment]
I'll have to take a look later today, that is usually a sign that mod id's changed somehow

Easy enough to fix, but this is why we wanted to beta test the map :D

**Comment by TyHen** [comment]
I know I am full of issues, and I am sorry.

**Comment by teagan75** [comment]
Don’t apologize 😁 This is why we wanted to give the map a test run

**Comment by TyHen** [comment]
It just seems I have found the problems, luckily they seem pretty easy fixes though!

**Comment by GreatOrator** [comment]
What have you found?  May be something I can adjust locally so no one else has to experience it.

**Comment by TyHen** [comment]
The doors were one, game difficulty was set to peaceful (easy fix), and the last one that I have seen is that the custom NPC's dialogue is cut through the screen as in you cant read the options until after you click on them. I've set the GUI to normal but it is still too large.

**Comment by GreatOrator** [comment]
Ahh, my bad, thought you meant you had found the fix for the problems :D  peaceful mode is certainly something I can adjust, though the dialog is odd, wait, no, I have seen that when the conversation response is long.  sadly that one is not something we can fix, it is the way Custom NPC's handles the dialog responses for formatting.

**Comment by TyHen** [comment]
Yes I kind of figured that with my experience with Custom NPC's, overall a great mod, but needs a little work.
